addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/onboard-account.cfn.yml

```diff
@@ -732,6 +741,19 @@ Resources:
       Tags:
         - Key: Name
           Value: Private Workspace Subnet
+  
+  # For an ALB - You must specify subnets from at least two Availability Zones.
+  PrivateWorkspaceSubnet2:
```

Comment:
Can we create two independent subnets just for ALB? Can you check if workspace subnets and ALB subnets?

Reply:
Done in ALB template